packages/clerk-js/src/ui/components/UserProfile/MfaBackupCodeCreatePage.tsx

```diff
@@ -27,11 +31,19 @@ export const MfaBackupCodeCreatePage = () => {
   }, []);
 
   if (card.error) {
-    return <ContentPage headerTitle={localizationKeys('userProfile.backupCodePage.title')} />;
+    return (
+      <ContentPage
```

Comment:
❓ How come we no longer use the ContentPage from '/userProfile'?
Do you think its better to explicitly pass        ` Breadcrumbs={UserProfileBreadcrumbs}` everywhere, instead of creating a component to abstract this prop?